validator/testcases/javascript/traverser.py

```diff
@@ -15,6 +15,7 @@
 IGNORE_POLLUTION = False
 POLLUTION_COMPONENTS_PATH = re.compile(r"/?components/.*\.jsm?")
 POLLUTION_EXCEPTIONS = set(["Cc", "Ci", "Cu", ])
+MAX_LINE_COMPLEXITY = 100
```

Comment:
@magopian Can you share the rationale here too so it's documented? We might want similar checks in the JS validator.